src/libs/ReportUtils.ts

```diff
@@ -2143,7 +2143,14 @@ function getGroupChatName(participants?: SelectedParticipant[], shouldApplyLimit
         return report.reportName;
     }
 
-    let participantAccountIDs = participants?.map((participant) => participant.accountID) ?? Object.keys(report?.participants ?? {}).map(Number);
+    let participantAccountIDs =
+        participants?.map((participant) => participant.accountID) ??
+        Object.keys(report?.participants ?? {})
+            .map(Number)
+            .filter((accountID) => {
+                const pendingMember = report?.pendingChatMembers?.find((member) => member.accountID === accountID.toString());
+                return !(pendingMember && pendingMember.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE);
+            });
```

Comment:
```suggestion
const pendingMemberAccountIDs = new Set(
    report?.pendingChatMembers
        ?.filter((member) => member.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE)
        .map((member) => member.accountID)
);

let participantAccountIDs =
    participants?.map((participant) => participant.accountID) ??
    Object.keys(report?.participants ?? {})
        .map(Number)
        .filter((accountID) => !pendingMemberAccountIDs.has(accountID.toString()));
```

`find` must not be an issue because the `pendingMembers` will likely be few in number. However, extracting to a set `pendingMemberAccountIDs` makes the code cleaner also IMO.